StructureDefinition/England-HealthcareService.xml

```diff
@@ -0,0 +1,307 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+    <id value="England-HealthcareService" />
+    <url value="https://fhir.nhs.uk/StructureDefinition/England-HealthcareService" />
+    <name value="EnglandHealthcareService" />
+	<title value="England HealthcareService" />
+    <status value="draft" />
+    <fhirVersion value="4.0.1" />
+    <kind value="resource" />
+    <abstract value="false" />
+	<version value="1.0.0" />
+    <type value="HealthcareService" />
```

Comment:
This looks different to other profiles. Missing contact details, description and purpose

Reply:
Copyright is also missing.